Examples/Cocoa/Swift/Sources/Swifty/Pin.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  Pin.swift
+//  Autogenerated by plank
+//
+//  DO NOT EDIT - EDITS WILL BE OVERWRITTEN
+//  @generated
+//
+
+import Foundation
+
+public struct Pin: Codable {
+    public var note: String
```

Comment:
Planning on using `var` here to take advantage of copy-on-write. Moreover, a struct assigned to a constant property will prevent the struct fields from being modified. Making them each `let` would make the API annoying to work with imo, since you would have to either create a builder API or instantiate a new struct with an initializer anytime you would want to change a field.

Good catch on the optionals. I'm about to run this against my own app next week, which will give me a chance to run it against these edge cases.